.github/workflows/beagle-bone-blue-binaries.yml

```diff
@@ -0,0 +1,101 @@
+name: beagle-bone-blue-binaries
+on:
+  push:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '30 5,17 * * *'
+jobs:
+  latest:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
```

Comment:
we don't need nodejs

Reply:
Right, that needs to be removed. Is still in there because I used an example as starting point for that file.